scale/workloads/np-load.yaml

```diff
@@ -0,0 +1,45 @@
+deploymentWorkload:
+- deploymentType: Deployment
+  lifecycleDuration: 30m0s
+  numDeployments: 500
+  numLifecycles: 0
+  numLabels: 10
```

Comment:
This is the only difference between this copy and the other block `deploymentType: Deployment`. The other block would get 3 labels (`createRandMap(16, 3)`) and this one would get 10. I am just wondering why there are two of them?

Reply:
100% agree. I think this can be solved by adding the fields that I commented above.